README.md

```diff
@@ -1,2 +1,70 @@
 # comp110-worksheet-C
+<<<<<<< HEAD
+Base repository for COMP110 worksheet C (Arturs Zingis)
+
+
+**Algorithm:**
+
+	procedure HASDUPLICATE(list)
+	let n be the length of list
+	for i = 0, 1,..., n - 1 do
+	for j = 0, 1,..., n - 1 do
+	if i =/ j and list[i] = list[j] then
+	return true
+	end if
+	end for
+	end for
+	return false
+	end procedure
+
+
+A) O(n). This algorithm sets two lists with i/j and for every item in the list, it takes away 1 from the length of the list and then checking if both lists are equal and if they are then return true and if not, return false. 
```

Comment:
This seems to be explaining the operation of the algorithm line by line -- what is the task it actually performs? It checks a list for duplicated elements.